### PR TITLE
76 map marker icons based on harbor status

### PR DESCRIPTION
**Pull request by artur-gajewski**

### CODE REVIEW

File: .lintstagedrc [hunk 1]
```diff
@@ -1,4 +1,3 @@
 {
-    "./**/*.js": ["yarn eslint --fix", "yarn prettier --write", "git add"],
-    "./**/*.{json,scss}": ["yarn prettier --write", "git add"],
+    "./**/*.{js,json,scss}": ["yarn prettier --write", "git add"],
```
Comment (Cifixie) [inline .lintstagedrc]: 👍

File: src/components/Map.js [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
 import React, { Component } from 'react';
-import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
-import { Container, Row, Col, Button } from 'reactstrap';
+import { Map, TileLayer } from 'react-leaflet';
 import L from 'leaflet';
+import MapMarker from './MapMarker';
 
-import fence from './common/icons/fence.svg';
-import plug from './common/icons/plug.svg';
+import HarborMatchSelected from './common/icons/harbor-match-chosen.svg';
```
Comment (Cifixie) [inline src/components/Map.js]: 😃 inconsistent naming, could be `BerthSelected` / `berth-selected.svg`

File: src/components/Map.js [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+import HarborMatchUnselected from './common/icons/harbor-unmatch.svg';
```
Comment (Cifixie) [inline src/components/Map.js]: 😃 inconsistent naming, could be `BerthUnSelected ` / `berth-unselected.svg`

File: src/components/Map.js [hunk 4]
```diff
@@ -41,48 +41,37 @@ type Props = any;
 
 export default class SimpleExample extends Component<Props, State> {
   state = {
-    lng: 25.066105,
+    lng: 24.93,
     lat: 60.18808,
-    zoom: 14
+    zoom: 11.47
```
Comment (Cifixie) [inline src/components/Map.js]: magic

File: src/components/Map.js [hunk 5]
```diff
@@ -41,48 +41,37 @@ type Props = any;
 
 export default class SimpleExample extends Component<Props, State> {
```
Comment (Cifixie) [inline src/components/Map.js]: 😏

File: src/components/MapMarker.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Marker, Popup } from 'react-leaflet';
+import { Container, Row, Col, Button } from 'reactstrap';
+import { List } from 'immutable';
+
+type Props = any;
```
Comment (Cifixie) [inline src/components/MapMarker.js]: We shouldn't allow `any`

File: src/components/MapMarker.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+export default class MapMarker extends Component<Props> {
+  static defaultProps = {
```
Comment (Cifixie) [inline src/components/MapMarker.js]: I don't think this does what you think it does 🤔 it isn't typing

File: src/components/MapMarker.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <strong>{berth.name.fi}</strong>
+                <div>{berth.street_address.fi}</div>
+                <div>{berth.municipality.fi}</div>
+                {selected ? (
+                  <Button color="secondary" onClick={() => onClick(berth.identifier)}>
+                    Valittu
```
Comment (Cifixie) [inline src/components/MapMarker.js]: Not translated

File: src/components/pages/BerthPage.js [hunk 9]
```diff
@@ -69,7 +69,8 @@ class BerthPage extends Component<Props> {
                   />
                   <BerthsOnMap
                     TabHeader={() => <span>Kartalla</span>}
-                    berths={filtered}
+                    berths={berths}
```
Comment (Cifixie) [inline src/components/pages/BerthPage.js]: unused

File: src/components/Map.js (hunk 4 above)
Comment (artur-gajewski) [reply src/components/Map.js]: It is based on carefully analyzing human eye preference and mathematically calculating visually best outcome.

File: src/components/Map.js (hunk 5 above)
Comment (artur-gajewski) [reply src/components/Map.js]: lol

**Pull request merged**